gunicorn/util.py

```diff
@@ -22,7 +22,10 @@
 import traceback
 import warnings
 
-import pkg_resources
+try:
+    import importlib.metadata as importlib_metadata
+except:
```

Comment:
`gunicorn\util.py:27:1: E722 do not use bare 'except'`
```suggestion
except Exception:
```